### PR TITLE
PROVIDER_ID_FORMAT references replacement.

### LINKED ISSUE #1471: Replace reference of POWERVS_PROVIDER_ID_FORMAT with PROVIDER_ID_FORMAT
/kind feature
/area provider/ibmcloud

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]

Power VS provider id format has been deprecated and removed via [PR](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/1409), [PR](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/1404). 

We are supposed to use `PROVIDER_ID_FORMAT` environmental variable or `--provider-id-fmt` flag. 

Need to update code and document to refer the same.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

### PR DESCRIPTION
**Pull request by kishen-v**
**What this PR does / why we need it**:
This PR contains the changes to point to the correct usage of PROVIDER_ID_FORMAT in the .md files of the CAPI book, updated references to keep documents inline with the intended usage.

Also contains a change in table's format and alignment, to adapt to a smaller space.

**Which issue(s) this PR fixes**
Fixes #1471 

**Special notes for your reviewer**:

/area provider/ibmcloud

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ cluster on IBM Cloud.
...
 This provider's versions are compatible with the following versions of Cluster API:
 
-|                                            | v1alpha4 (v0.4) | v1beta1 (v1.x) |
-|--------------------------------------------|-----------------|----------------|
-| IBMCloud Provider v1alpha4 (v0.1.x)        | ✓               |                |
-| IBMCloud Provider v1beta1 (v0.2.x, v0.3.x) |                 | ✓              |
-| IBMCloud Provider v1beta2 (v0.4.x, v0.5.x, v0.6.x, main) |                 | ✓              |
+|                                         |Cluster API v1alpha4 (v0.4) |Cluster API v1beta1 (v1.x) |
+|:----------------------------------------|:---------------:|:--------------:|
+| CAPI v1alpha4 (v0.1.x)                  | ✓               |                |
```
Comment (mkumatag) [inline README.md:32]: why is this CAPI? this is suppose to be CAPIBM

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kishen-v) [reply README.md]: @mkumatag, I have made the required changes. 
Thanks!

File: docs/book/src/topics/powervs/external-cloud-provider.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 - To deploy a PowerVS workload cluster with IBM PowerVS external [cloud provider](https://kubernetes.io/docs/concepts/architecture/cloud-controller/), create a cluster configuration with the [external cloud provider template](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/templates/cluster-template-powervs-cloud-provider.yaml)
 - The [external cloud provider template](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/templates/cluster-template-powervs-cloud-provider.yaml) will use [clusterresourceset](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-resource-set.html) and will create the necessary config map, secret and roles to run the cloud controller manager
-- As a prerequisite set the `powervs-provider-id-fmt` [flag](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/64c9e1d17f1733c721f45a559edba3f4b712bcb0/main.go#L220) with value v2
+- As a prerequisite set the `provider-id-fmt` [flag](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/64c9e1d17f1733c721f45a559edba3f4b712bcb0/main.go#L220) with value v2
```
Comment (Karthik-K-N) [inline docs/book/src/topics/powervs/external-cloud-provider.md:8]: May be we need to update the link as well. https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/64c9e1d17f1733c721f45a559edba3f4b712bcb0/main.go#L220

**Review by Karthik-K-N — COMMENTED** [review]
Lets validate links otherwise looks good to me.

File: README.md (hunk 1 above)
Comment (kishen-v) [reply README.md]: @mkumatag, a miss from my end. Will update accordingly. 
Thanks!

File: docs/book/src/topics/powervs/external-cloud-provider.md (hunk 2 above)
Comment (kishen-v) [reply docs/book/src/topics/powervs/external-cloud-provider.md:8]: Agree, the links are broken. I shall work on them.

Comment (kishen-v) [reply docs/book/src/topics/powervs/external-cloud-provider.md]: Updates with the changes from the latest code. Thank you. :)

**Review by mkumatag — APPROVED** [review]
/lgtm

/hold 

for @Karthik-K-N to review

**Pull request merged**